packages/mui-joy/src/Link/Link.test.js

```diff
@@ -0,0 +1,326 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import { act, createRenderer, fireEvent, describeConformance } from 'test/utils';
+import Typography from '@mui/joy/Typography';
+import Link, { linkClasses as classes } from '@mui/joy/Link';
+
+function focusVisible(element) {
+  act(() => {
+    element.blur();
+    document.dispatchEvent(new window.Event('keydown'));
+    element.focus();
+  });
+}
+
+describe('<Link />', () => {
+  const { render } = createRenderer();
+
+  describeConformance(<Link href="/">Home</Link>, () => ({
+    classes,
+    inheritComponent: Typography,
+    render,
```

Comment:
You will need to import `ThemeProvider` from joy.

```suggestion
    render,
    ThemeProvider,
```